ops/pebble.py

```diff
@@ -3098,7 +3160,7 @@ def get_notices(
         *,
         users: Optional[NoticesUsers] = None,
         user_id: Optional[int] = None,
-        types: Optional[Iterable[Union[NoticeType, str]]] = None,
+        types: Optional[Iterable[NoticeType]] = None,
         keys: Optional[Iterable[str]] = None,
     ) -> List[Notice]:
         """Query for notices that match all of the provided filters.
```

Comment:
I wonder if this will break any users.

Reply:
There's not a single user of .get_notices() yet.
Source: charm-analysis and github search.